packages/ocean-react/src/Drawer/Drawer.tsx

```diff
@@ -10,37 +10,87 @@ interface DrawerProps {
   onDrawerClose?(event: React.MouseEvent | React.KeyboardEvent): void;
   overlayClose: () => void;
   headerIcon?: React.ReactNode;
-  iconAlignment?: string;
+  align?: 'right' | 'left';
+  iconAlignment?: 'right' | 'left';
+  anchorEl?: RefObject<HTMLDivElement> | null;
 }
 
 const Drawer = ({
   children,
   open,
   onDrawerClose,
   overlayClose,
+  align = 'right',
   headerIcon = <XOutline />,
   iconAlignment = 'right',
-}: DrawerProps): React.ReactElement => (
-  <>
+  anchorEl,
+}: DrawerProps): React.ReactElement => {
+  const drawerRef = useRef<HTMLDivElement>(null);
+
+  const getAnchorPosition = useCallback(() => {
+    if (!anchorEl?.current) return null;
+
+    const position = anchorEl.current.getBoundingClientRect();
+
+    if (align === 'right') {
+      const size = window.innerWidth - position.left;
```

Comment:
Acredito que a utilização da variável global `window` irá impactar caso em algum momento sugir a necessidade de usar o Ocean em um framework/ferramenta SSR. Isso aconteceu uma vez comigo quando tentei setar o Ocean com o Next. Talvez teremos que pensar alguma estratégia no futuro para contornar isso.

https://upmostly.com/next-js/how-to-fix-window-is-not-defined-in-next-js

Reply:
Creio que nesse ponto do código não seja um problema, pois mais acima tem uma validação se o elemento ref já está criado. E uma vez que ele criado já existiria o window.

Reply:
Inclusive a função `getBoundingClientRect` logo acima por baixo dos panos também usa o window no seu cálculo.